Sources/SwiftKuery/Column.swift

```diff
@@ -172,5 +230,30 @@ public class Column: Field, IndexColumn {
         }
         return result
     }
+    
+    //MARK: Column Expressions
+
+    /**
+     Function to return a copy of the current `Column` instance with the given name as its alias.
+     This is equivelent to the SQL AS operator.
+     ### Usage Example: ###
+     In this example, a `PersonTable` (as defined at the top of this class) instance is created which contains a `Column`.
+     An alias for this `Column` instance is then created and its alias printed. 
+     ```swift
+     let personTable = PersonTable()  // PersonTable() is a previously defined `Table` class
+     let aliasColumn = personTable.name.as("new name")
+     print(String(describing: aliasColumn.alias))
+     //Prints Optional("new name")
+     ```
+     */
+    /// - Parameter newName: A String containing the alias for the column.
+    /// - Returns: A new Column instance with the alias.
+    public func `as`(_ newName: String) -> Column {
+        let new = Column(name, type, length: length, primaryKey: isPrimaryKey, notNull: isNotNullable, unique: isUnique, defaultValue: defaultValue, collate: collate)
+        new.alias = newName
+        new._table = table
+        return new
+    }
+    
```

Comment:
I think we should raise a further issue to create examples for the other SQL methods...  like(), notLike, between etc even if we only create one example for each type.

Reply:
Andy, did you raise an issue for this?